CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 0.43.0
 
+Added support for additional volumes in CRDs:
+* Users can specify volumes and volumeMounts fields in the Kafka, KafkaConnect, KafkaBridge, KafkaMirrorMaker2, EntityOperator, CruiseControl, KafkaExporter, and Zookeeper CRDs.
+* Supported volume types include Secret, ConfigMap, EmptyDir, and PersistentVolumeClaims.
+* All additional mounted paths is mounted inside /mnt to ensure backwards compatibility for the evolution of kafka and this operator.
```

Comment:
```suggestion
* All additional mounted paths are mounted under `/mnt` to ensure backwards compatibility for the evolution of kafka and this operator.
```

Reply:
Can we really claim this ensures backwards compatibility for the evolution of components external to strimzi (i.e. kafka). The directories used by components from other communities is surely a matter for those communities.

This limitation also provides no guarantees for the cluster operator as users can today mount anything they want to any location they want on the cluster operator.

Reply:
We normally keep the CHANGELOG very simple. It should not replace the docs. So I think all you need there is the following:

```
* Added support for additional volumes in CRDs.
```

The rest can be left for the docs. And even the docs don't really need explanation why were things done as they were. That can be left for the proposal.